src/houghtransform.jl

```diff
@@ -0,0 +1,102 @@
+
+#function to compute local maximum lines with values > threshold and return a vector containing them
+function findlocalmaxima(accumulator_matrix::Array{Integer,2},threshold::Integer)
+    validLines = Vector{CartesianIndex}(0)
+    for val in CartesianRange(size(accumulator_matrix))
+        if  accumulator_matrix[val] > threshold &&
+            accumulator_matrix[val] > accumulator_matrix[val[1],val[2] - 1] &&
```

Comment:
I see. If you really require custom behavior, it's fine to have a private `findlocalmaxima` but obviously don't export it or import it from Images.